utils/callData/registrationCalls.ts

```diff
@@ -221,7 +221,12 @@ export const getFreeRegistrationCalls = (
     {
       contractAddress: process.env.NEXT_PUBLIC_DOMAIN_GIFT_CONTRACT as string,
       entrypoint: "get_free_domain",
-      calldata: [newTokenId, encodedDomain, signature, txMetadataHash].flat(),
+      calldata: [
+        numberToStringHex(newTokenId),
+        numberToStringHex(encodedDomain),
+        signature.map((s) => numberToStringHex(s)),
+        txMetadataHash,
+      ].flat(),
```

Comment:
You you change here ?

Reply:
Same as here https://github.com/starknet-id/app.starknet.id/pull/841#discussion_r1655224540